hwi-qt.spec

```
@@ -11,7 +11,7 @@ def get_libusb_path():
     if platform.system() == 'Darwin':
         proc = subprocess.Popen(['brew', '--prefix', 'libusb'], stdout=subprocess.PIPE)
         prefix = proc.communicate()[0].rstrip().decode()
-        return os.path.join(prefix, "/lib/libusb-1.0.dylib")
+        return os.path.join(prefix + "/lib/libusb-1.0.dylib")
```

The correct way of fixing this is:

```suggestion
        return os.path.join(prefix, "lib", "libusb-1.0.dylib")
```

---

I don't mind how it is fixed as long it is fixed because currently it is broken and can be replicated with a macbook installing libusb through brew.

---

I prefer this to be implemented using the correct join syntax as suggested by @prusnak.